alignment.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>
+        Alignment
+    </title>
+    <link rel="stylesheet" href="css/alignment.css">
+    <style>
+        .system-fonts {
+          font-family: sans-serif;
+        }
+      </style>
+</head>
+<body>
+    <header>
+        <nav>
+          <ul>
+            <li><a href="/index.html">Web Fonts</a></li>
+            <li><a href="./common/history.html">History</a></li>
+            <li><a href="./common/indents.html">Indents</a></li>
+            <li class="current">Alignment</li>
+            <li><a href="./common/spacing.html">Spacing</a></li>
+            <li><a href="./common/line-length.html">Line Length</a></li>
+          </ul>
+        </nav>
+  
+        <div class="hero">
+          <h1>Text Alignment</h1>
+        </div>
+      </header>
+
+      <main class="container">
+        <section class="section section--gray">
+          <h2>Left Alignment</h2>
+          <p>
+            Left-aligned text is the standard because it gives the reader a
+            vertical anchor to jump back to on every line. You should use it for
+            most of your body text (unless you have a very good reason not to).
+          </p>
+        </section>
+        <section class="section section--gray system-fonts">
+          <h2>Center Alignment</h2>
+          <p class="centrify">
+            Center-aligned text doesn’t have that anchor, so it’s easier for the
+            eye to get lost when it tries to jump to the next line. Reserve it for
+            special kinds of content, like poems, lyrics, and headings.
+          </p>
+        </section>
+        <section class="section section--gray system-fonts">
+            <h2>Right Alignment</h2>
+            <p>
+                Only use right-aligned text when it makes sense for the layout. For
+                example, a caption for an image:
+            </p>
+        <div class="ligaturediv">
```

Comment:
I don't think we've used the `figure` tag  but this would actually be a good use case for it compared to divs. [https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure](url)

So the markup may look something like:
```
       <figure>
          <figcaption>
            These letters are touching. This is called a ligature.
          </figcaption>
          <img src="../img/ligature.svg" />
        </figure>
```